cluster-autoscaler/core/scaleup/orchestrator/async_initializer.go

```diff
@@ -155,6 +155,13 @@ func (s *AsyncNodeGroupInitializer) InitializeNodeGroup(result nodegroups.AsyncN
 		return
 	}
 	klog.Infof("Initial scale-up succeeded. Scale ups: %v", scaleUpInfos)
+	scaleUpStatus := &status.ScaleUpStatus{
```

Comment:
Why isn't this using `emitScaleUpStatus` function?

Reply:
I just wanted to omit passing explicit nil as error and then checking on it in `emitScaleUpStatus`. Though it wasn't a strong inclination so changed.